### PR TITLE
show filters regardless of past page history

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?216599

@biyeun

### CODE REVIEW

File: corehq/apps/reports/static/reports/javascripts/reports.config.js [hunk 1]
```diff
@@ -113,27 +112,11 @@ var HQReport = function (options) {
...
-        }
-        $(self.filterAccordion).addClass($.cookie(self.toggleFiltersCookie));
-
-        if ($.cookie(self.toggleFiltersCookie) == 'in') {
-            $(self.toggleFiltersButton).button('close');
-        } else {
-            $(self.toggleFiltersButton).button('open');
-        }
+        $(self.filterAccordion).addClass('in');
+        $(self.toggleFiltersButton).button('close');
```
Comment (nickpell) [inline corehq/apps/reports/static/reports/javascripts/reports.config.js]: Not sure what ```$(self.toggleFiltersButton).button('close');``` does, but was called when ```$.cookie(self.toggleFiltersCookie) == 'in'```

**Comment by czue** [comment]
looks good and :+1: for :skull: :cookie: 

will let @biyeun take a closer look since she's got more context

**Comment by snopoke** [comment]
Nice, this has been a long time coming! :skull:

File: corehq/apps/reports/static/reports/javascripts/reports.config.js (hunk 1 above)
Comment (biyeun) [inline corehq/apps/reports/static/reports/javascripts/reports.config.js]: It just means show the text state for that button named 'close'. http://getbootstrap.com/javascript/#buttons

**Pull request merged**